geospatial/include/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+add_subdirectory(gz)
+install(DIRECTORY ignition DESTINATION ${IGN_INCLUDE_INSTALL_DIR_FULL})
```

Comment:
`geospatial` is a new component from Garden, so we don't need to add `ignition` headers and can go straight to `gz`, because no downstream users should be using it yet anyway.

One caveat is that our libraries are already using it, so one strategy we could take would be to keep `ignition/geospatial` while we're updating all libraries, and then remove it afterwards.

Reply:
Might be neater to remove it during the namespace (or maybe the source) migration? I'm not sure :o

Reply:
I think any time after the header migration for all libraries should work, because none of the libraries should be using them anymore. So I'm fine either way. It's probably good to do it as a standalone PR for visibility. We just shouldn't forget to do it.